recipes/send-message/package.json

```diff
@@ -0,0 +1,16 @@
+{
+  "name": "@whispir-recipes/send-message",
+  "version": "1.0.0",
+  "description": "",
+  "main": "index.js",
+  "scripts": {
+    "start": "npx ts-node index.ts",
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "keywords": [],
+  "author": "",
+  "license": "ISC",
+  "dependencies": {
+    "dotenv": "^16.0.3"
```

Comment:
Once published, we'll need to add `whispir` as a dependency to all the recipes packages.